GSoC-2025/BrainGlobe: Improve cellfinder's classification algorithm(Mennat Allah).md

```diff
@@ -51,33 +55,32 @@ Deep learning has evolved with modern architectures like **Vision Transformer (V
 - **Post-exams:** 30+ hours per week dedicated to the project.
 
 
-| **Week Number**  | **Tasks**  |
-|-----------------|----------------------------------------------------------------------------------------------------------------------------------|
-| **Community Bonding Period** | Engage with the **cellfinder** community, study existing code, and discuss project goals with mentors. Gather feedback to refine the implementation approach. |
-| **Week 1-3**  | Research different deep learning algorithms. Communicate with mentors, study various architectures, and finalize the most suitable algorithm for **cellfinder** classification. I think I will be using Keras, as it's already used in the implementation of resnet model in the cellfinder algorithm and it aligns with my skills.  |
-| **Week 4-6**  | Implement the selected deep learning model. Define architecture, layers, and optimization techniques. Begin developing and integrating testing methods, test the model locally on a small dataset and  Seek feedback from mentors regularly. |
-| **Week 7-9**  | Conduct thorough **testing** and refine the model based on results. Document the implementation process, including model selection and evaluation techniques. |
-| **Week 10-11** | Finalize testing and make necessary refinements. Freeze the model's architecture and begin integrating findings into the final documentation and blog post. |
-| **Week 12**   | Complete final documentation and submit the blog post. Address any final feedback and ensure that the project meets all deliverables. If time permits, explore stretch goals. |
+| **Week Number** | **Tasks** |
+|-----------------|-------------------------------------------------------------------------------------------------------------------------------------------|
+| **Community Bonding Period** | Engage with the **cellfinder** community, study existing code, and discuss project goals with mentors. Discuss the dataset with the mentors and understand how the current model is trained and used. Study cellfinder's preprocessing pipeline to adapt B0's input requirements. |
+| **Week 1-3** | In this period, I will start implementing the EfficientNet architecture using TensorFlow/Keras. The implementation will utilize transfer learning from ImageNet pre-trained weights, which will be fine-tuned on neuroanatomical cell datasets. Then, initial testing will be conducted using a small subset of the data to ensure compatibility with the workflow. |
```

Comment:
It should be pytorch or keras (with a pytorch backend) to match the existing code